src/schema/graphql-base.ts

```diff
@@ -242,6 +242,12 @@ export const DIRECTIVES: DocumentNode = gql`
 
     "Annotates a field so it can be used within restrictions of a permission profile"
     directive @accessField on FIELD_DEFINITION
+
+    """
+    Mark this field as a technical field. This information can be queried and used to
+    hide the field from UIs.
```

Comment:
wouldn't use the term "technical" here - we settled on `hidden` because it's orthogonal to being "technical".

Maybe "Marks this field as hidden in the meta schema so it will not be listed in generic UIs"?

Reply:
I copied your suggestion since I do not have a strong opinion on this.

I tried to avoid writing something with `generic` in it, because it is a really specific use-case that other users of cruddl probably do not have (correct me if I am wrong, or we should not care about this 😅 )

Reply:
> I tried to avoid writing something with generic in it, because it is a really specific use-case that other users of cruddl probably do not have (correct me if I am wrong, or we should not care about this 😅 )

If they don't have a use case for generic UIs, they also don't have a use case for `@hidden` I'd say